src/components/overviewPanel/css/OverviewPanel.css

```diff
@@ -22,12 +22,17 @@
 
 @font-face {
     font-family: 'RobotoSlab';
-    src: url('../../../lib/roboto/fonts/Roboto-Slab-Bold.woff');
     font-weight: bold;
+    src: url('../../../lib/roboto/fonts/Roboto-Slab-Bold.woff');
 }
 
-
 .fl-overviewPanel-container {
+    font-family: "OpenSans", "Helvetica Neue", "Helvetica", Arial, sans-serif;
+    font-size: 1em;
+    height: 100%;
+    left: 0;
```

Comment:
It seems that the reordering of the properties has moved some positioning properties away from the comment. For example `left: 0;` is now above the comment and `position: fixed;` and `top: 0;` are below. I think it would make sense to move the comment above the declaration and reword as necessary. Another option would be to remove the comment completely if you don't find it necessary. I'm open to other options as well.